examples/catalyst_contrastive/common.py

```diff
@@ -0,0 +1,11 @@
+# flake8: noqa
+from catalyst import dl
+
+
+class ContrastiveRunner(dl.SupervisedRunner):
```

Comment:
for future development,
what do you think about placing it under `catalyst/runners/contrastive.py`?